src/ui/reducers/timeline.ts

```diff
@@ -15,8 +16,9 @@ function initialTimelineState(): TimelineState {
     focusRegion: getPausePointParams().focusRegion,
     focusRegionBackup: null,
     displayedFocusRegion: null,
-    hoverTime: null,
+    hoveredEvent: null,
```

Comment:
:question: Can we just make this a `TimeStampedPoint`?  We can use this same mechanism with things like the React Timeline panel too, which doesn't use the `ReplayEvent` type, and all we actually care about here _is_ the `ExecutionPoint` + time anyway.